### PR TITLE
M0078DDG-294 Update SSL page

### PR DESCRIPTION
**Pull request by balix-microej**
The SSL page now mentions support of mutual authentication.

### CODE REVIEW

**Comment by balix-microej** [comment]
> 
> 
> Seems that `LLNET_SSL_X509_CERT_impl.h` is no more avalaible in main stream pack 9.2.1 => delete it ?

OK done.

File: PlatformDeveloperGuide/ssl.rst [hunk 1]
```diff
@@ -9,7 +9,8 @@ Principle
 SSL (Secure Sockets Layer) library provides APIs to create and establish
 an encrypted connection between a server and a client. It implements the
 standard SSL/TLS (Transport Layer Security) protocol that manages client
-or server authentication and encrypted communication.
+or server authentication and encrypted communication. Mutual authentication
```
[the remark this answers is not in the document]
Comment (balix-microej) [reply PlatformDeveloperGuide/ssl.rst:12]: Is there an ESR specification for SSL ?

File: PlatformDeveloperGuide/ssl.rst [hunk 2]
```diff
@@ -9,7 +9,8 @@ Principle
...
+is supported since MicroEJ SSL 2.1.0.
```
[the remark this answers is not in the document]
Comment (balix-microej) [reply PlatformDeveloperGuide/ssl.rst:13]: Done.

File: PlatformDeveloperGuide/ssl.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (frivieremicroej) [reply PlatformDeveloperGuide/ssl.rst:12]: No SSL module is derived from OpenJDK

[the remark this answers is not in the document]
Comment (balix-microej) [reply PlatformDeveloperGuide/ssl.rst:12]: OK that should be good now.

**Pull request merged**